commands/peers/commands.go

```diff
@@ -0,0 +1,40 @@
+// Package peercommands implements the peer management commands
+package peercommands
+
+import (
+	"github.com/gluster/glusterd2/rest"
+)
+
+// Command is a holding struct used to implement the GlusterD Command interface
+type Command struct {
+}
+
+// Routes returns command routes. Required for the Command interface.
+func (c *Command) Routes() rest.Routes {
+	return rest.Routes{
+		rest.Route{
+			Name:        "GetPeer",
+			Method:      "GET",
+			Pattern:     "/peers/{peerid}",
```

Comment:
Getting the peer entry from the store by a peer name is *possible* if we can retrieve all the entries using GetPeers and then matching the peer name from those records. However I'd still like to have the key as name and retrieving the peer based on IDs can follow the approach discussed here. Let me know your thoughts. in volume's store-utils key is name, not the id at this moment.